daemon/daemon_main.go

```diff
@@ -481,6 +482,9 @@ func init() {
 	flags.StringSlice(option.Labels, []string{}, "List of label prefixes used to determine identity of an endpoint")
 	option.BindEnv(option.Labels)
 
+	flags.Bool(option.EnableKubeProxyFree, false, "Enable kube-proxy replacement in Cilium datapath (BPF)")
```

Comment:
Maybe add something "Enable kube.... (eBPF) if supported"

Reply:
Maybe: `Enable or partially enable kube-proxy BPF replacement in Cilium datapath if supported by the kernel`